packages/cli/src/__tests__/commands/Studio.test.ts

```diff
@@ -69,7 +69,7 @@ describe('studio with alternative urls and prisma://', () => {
   })
 
   testIf(process.platform !== 'win32')('queries work if url is prisma:// via the mini-proxy', async () => {
-    process.env.DATABASE_URL = `${process.env.TEST_POSTGRES_URI}-${Date.now()}-studio`
+    process.env.DATABASE_URL = process.env.TEST_POSTGRES_URI!.replace('tests', `tests-${Date.now()}-studio`)
```

Comment:
Could we use `PRISMA_DB_NAME` as a keyword instead?

Reply:
Very true